scripts/exregional_run_prepstart.sh

```diff
@@ -303,7 +303,8 @@ else
 #   So the defination of restart_prefix needs a "." at the end.
 #
   if [ ${cycle_subtype} == "spinup" ] ; then
-    restart_prefix="${YYYYMMDD}.${HH}00${DT_ATMOS}."
+    #restart_prefix="${YYYYMMDD}.${HH}00${DT_ATMOS}."
```

Comment:
If line 306 is no longer needed, suggest removing it.

Reply:
save this for future development.